### DISCUSSION TITLE
error gencase

### DISCUSSION THREAD

**Discussion by GiuliaClai** [DualSPHysics/DualSPHysics#233] [DUALSPHYSICS_V5.4_BETA]
I did step by step the CaseDambreak with DualSPHysics_v5.4_BETA and releted DesignSPHysics. When I  Run GenCase I got this message but I don't know what to do. I thank anybody who can give a suggestion.

  Traceback (most recent call last):
   File "C:\Users\user\AppData\Roaming\FreeCAD\Mod\DesignSPHysics\mod\widgets\dock\dock_pre_processing_widget.py", line 270, in on_execute_gencase
    total_particles = int(total_particles_text[total_particles_text.index(": ") + 2:])
 ValueError: invalid literal for int() with base 10: '16,001'

**Comment by imestevez** [comment]
Hello, pease check that your computer is configured to use the comma character for thousands and the dot for decimals.

**Reply by GiuliaClai** [reply]
hallo, as you suggested I changed the comma character for thousands and the dot for decimals, but the program gives me the same mistake.
Any other suggestion? My PC has the same configuration as another from a friend on which I did not have the same problem. It sounds strange to me, but coul it be a mother board problem?
My PC is WIN10 and the one on which I coud proceed is a WIN11, and mine cannot be upgraded to WIN11

**Comment by GiuliaClai** [comment]
I found the problem it was a FreeCad problem, I had to upgrade te sw. Thanks
